base-notebook/README.md

```diff
@@ -29,6 +29,12 @@ Take note of the authentication token included in the notebook startup log messa
 
 The Docker container executes a [`start-notebook.sh` script](./start-notebook.sh) script by default. The `start-notebook.sh` script handles the `NB_UID`, `NB_GID` and `GRANT_SUDO` features documented in the next section, and then executes the `jupyter notebook`.
 
+You can launch [JupyterLab](https://github.com/jupyterlab/jupyterlab) by setting `JUPYTER_ENABLE_LAB`:
+
+```
+docker run -it --rm -e JUPYTER_ENABLE_LAB=1 --rm -p 8888:8888 jupyter/base-notebook
```

Comment:
Later in the READMEs we document the following way to start JupyterLab instead of Notebook in these images with:

```
docker run -it --rm -p 8888:8888 jupyter/base-notebook start.sh jupyter lab
```

I'm thinking we should extend the start-singleuser script to accept an alternative command like start.sh does like so:

```
# defaults to the original behavior of running jupyterhub-singleuser
docker run -it --rm -p 8888:8888 jupyter/base-notebook start-singleuser.sh 
# overrides the command to run jupyter labhub instead
docker run -it --rm -p 8888:8888 jupyter/base-notebook start-singleuser.sh jupyter labhub
```

That way we can avoid introducing a env var that we have to maintain as a special feature of the stack images. What do you think?

Reply:
That works too. I still think env var is somewhat less intrusive though.  Image you have a jupyterhub with multiple images that some have lab installed, and some might be older ones.  Setting the env var now suggests using jupyterlab whenever it is available.

If we require the spawner to specify singleuser_cmd, people using formspawner/profilesspawner will need to carefully select the appropriate start command for individual image.

Reply:
Thanks for the clarification about the use case. I think the env var makes sense, now that I understand some images might have the env var set while others may not.